### ISSUE TITLE
project still on hold I am frustrated

### ISSUE THREAD

**Issue by Alino** [xolvio/meteor-coverage#6]
Hi, I am really waiting for this, can I help you somehow?

**Comment by samhatoum** [comment]
Thanks for offering!

We have the know-how but it's not possible yet with the current meteor build system. Waiting on the MDG to allow us to process files before they've been built.

**Comment by Sanjo** [comment]
Well, you could post a bounty https://www.bountysource.com/.

I don't have this feature currently on my work roadmap. I personally find code test coverage as metric not that useful.

**Comment by Alino** [comment]
Thanks for info both of you. That bountysource site is really interesting. Hopefully the MDG team will implement better build system soon.

**Comment by RomFouq** [comment]
Isn't the new Build API the solution to pre-process files during the build ?
https://github.com/meteor/meteor/wiki/Build-Plugins-API

**Comment by samhatoum** [comment]
yes it's now possible, but we need the time to make it work! It seems there's enough demand for this feature. A bounty source would be a good idea for this I think
